monai/networks/blocks/fcn.py

```diff
@@ -0,0 +1,227 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Type
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchvision import models
```

Comment:
Hi @yiheng-wang-nv please make torchvision optional using `optional_import`. it'll be something like:
```python
from monai.utils import exact_version, optional_import
models, _ = optional_import("torchvision", "0.5.0", exact_version, "models")
```

Reply:
Sorry, I wrongly clicked the request re-review button, plz ignore that. Let me think about how to use the pretrained model in a easier way